src/entities/group/group.service.ts

```diff
@@ -0,0 +1,70 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import { GroupModel, UserGroupModel } from '../../db/models';
+import { GroupDto, GroupInterface } from './group.interface';
+import { sequelizeConnection } from '../../db';
+
+class GroupService {
+  async getById(groupId: string) {
+    return await GroupModel.findByPk(groupId);
+  }
+
+  async getAll(limit: number) {
+    return await GroupModel.findAll({
+      limit,
+    });
+  }
+
+  async create(groupData: GroupDto): Promise<GroupModel> {
+    const { name, permissions } = groupData;
+
+    const newGroup = new GroupModel();
+    newGroup.id = uuidv4();
```

Comment:
You can use `defaultValue: DataTypes.UUIDV4` in your model and in that case you can get rid of `uuid ` dependency. But it is not critical I think